cli/app.go

```diff
@@ -1957,6 +1975,11 @@ var app = &cli.App{
 											Usage:    "version of the ML training script to use for training.",
 											Required: true,
 										},
+										&cli.StringFlag{
+											Name:     mlTrainingFlagContainerVersion,
+											Usage:    "ml training container version to use. Must be one of the supported types found by calling ListSupportedContainers",
```

Comment:
[opt] "ml training container version to use. Must be one of the supported container _names_ found by calling ListSupportedContainers"